traefik/templates/_helpers.tpl

```diff
@@ -47,6 +47,19 @@ The name of the service account to use
 {{- default (include "traefik.fullname" .) .Values.serviceAccount.name -}}
 {{- end -}}
 
+{{/*
+The name of the ClusterRole and ClusterRoleBinding to use.
+Adds the namespace to name to prevent duplicate resource names when there
+are multiple namespaced releases with the same release name.
+*/}}
+{{- define "traefik.clusterRoleName" -}}
+{{- if .Values.rbac.namespaced -}}
+{{- template "traefik.fullname" . -}}-{{ .Release.Namespace }}
+{{- else -}}
+{{- template "traefik.fullname" . -}}
+{{- end -}}
+{{- end -}}
```

Comment:
```suggestion
{{- define "traefik.clusterRoleName" -}}
{{- template "traefik.fullname" . -}}-{{ .Release.Namespace }}
{{- end -}}
```

It's quite unexpected to have a different name when you enable a feature. 
And this issue can also happens without `rbac.namespaced`.